entrypoint.sh

```diff
@@ -1,6 +1,6 @@
 #! /bin/bash
 input_paths="$1"
-logging_mode="$2"
+severity_mode="$2"
```

Comment:
We can set the default value ('style') for the severity mode right here by using
severity_mode="${2-style}"
This way we know that it always has a value and when it comes to checking its value we don't need the nested if on line 17. If the value is given and is not one of those 4 valid options, we echo the error and set the severity_mode back to style.

Reply:
Wow, I never knew that. I'll make that change. Thanks